src/lib/features/instance-stats/instance-stats-service.ts

```diff
@@ -287,34 +288,19 @@ export class InstanceStatsService {
         };
     }
 
-    getStatsSnapshot(): InstanceStats | undefined {
-        return this.snapshot;
-    }
-
     async getLabeledAppCounts(): Promise<
-        { range: TimeRange; count: number }[]
+        Partial<{ [key in TimeRange]: number }>
     > {
-        return [
-            {
-                range: 'allTime',
-                count:
-                    await this.clientInstanceStore.getDistinctApplicationsCount(),
-            },
-            {
-                range: '30d',
-                count:
-                    await this.clientInstanceStore.getDistinctApplicationsCount(
-                        30,
-                    ),
-            },
-            {
-                range: '7d',
-                count:
-                    await this.clientInstanceStore.getDistinctApplicationsCount(
-                        7,
-                    ),
-            },
-        ];
+        const [t7d, t30d, allTime] = await Promise.all([
+            this.clientInstanceStore.getDistinctApplicationsCount(7),
+            this.clientInstanceStore.getDistinctApplicationsCount(30),
+            this.clientInstanceStore.getDistinctApplicationsCount(),
+        ]);
+        return {
+            '7d': t7d,
+            '30d': t30d,
```

Comment:
Rename t7d to 7d and t30d to 30d?

Reply:
I'd love to, but it's not syntactically correct. Identifiers can't start with numbers.